src/stories/shared/StatusText.stories.tsx

```diff
@@ -23,7 +24,7 @@ const statusTextMeta: Meta<typeof StatusText> = {
     },
   },
   args: {
-    status: 'success',
+    status: StatusTypes.SUCCESS,
```

Comment:
As, with actions, these should be CamelCase as well

```suggestion
    status: StatusTypes.Success,
```